### PR TITLE
Add importer for ISO 9660 files

### PR DESCRIPTION
**Pull request by ZetaTwo**
This importer adds support for "regular" iso disk files. One issue is that it uses a fairly old package: https://pkg.go.dev/github.com/hooklift/iso9660 If this is an issue it's possible to change it to shell out to `mount` etc like the Windows importer does.

### CODE REVIEW

File: importers/iso9660/iso9660.go [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+	}
+
+	// Step 6: Extract file contents
+	if _, err := io.Copy(ff, freader); err != nil {
+		fmt.Errorf("error while extracting file data: %v", err)
+	}
+	return nil
+}
+
+// ID returns non-unique zip Archive ID.
```
Comment (meeehow) [inline importers/iso9660/iso9660.go:150]: Please change to ISO

File: importers/iso9660/iso9660.go [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+func (a *Archive) RepoName() string {
+	return RepoName
+}
+
+// RepoPath returns repository path.
+func (a *Archive) RepoPath() string {
+	return a.repoPath
+}
+
+// LocalPath returns local path to a zip Archive .tar.gz file.
```
Comment (meeehow) [inline importers/iso9660/iso9660.go:165]: Please change that, same below

File: importers/iso9660/iso9660.go [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+	_, err = f.ReadAt(footer, fileInfo.Size()-int64(chunkSize))
+	if err != nil {
+		return "", err
+	}
+
+	a.quickSha256hash = fmt.Sprintf("%x", sha256.Sum256(append(header, footer...)))
+	return a.quickSha256hash, nil
+}
+
+// NewRepo returns new instance of zip repository.
```
Comment (meeehow) [inline importers/iso9660/iso9660.go:224]: ISO repository

**Review by meeehow — CHANGES_REQUESTED** [review]
Couple of comments, mostly on naming

**Comment by ZetaTwo** [comment]
Think it should be all updated now.

**Pull request merged**